lib/home/schedule/schedule.dart

```diff
@@ -1,37 +1,36 @@
 import 'package:flutter/material.dart';
 import 'package:provider/provider.dart';
-import 'package:thc/models/navigator.dart';
-import 'package:thc/models/theme.dart';
-import 'package:thc/models/user.dart';
-import 'package:thc/views/home/home_screen.dart';
-import 'package:thc/views/manage_schedule/manage_schedule.dart';
+import 'package:thc/home/home_screen.dart';
+import 'package:thc/home/schedule/edit_schedule/schedule_editor.dart';
+import 'package:thc/utils/navigator.dart';
+import 'package:thc/utils/theme.dart';
+import 'package:thc/utils/user.dart';
 
 class Schedule extends StatelessWidget {
   const Schedule({super.key});
 
   @override
   Widget build(BuildContext context) {
+    final colors = context.colorScheme;
+
+    Widget? editButton;
+    if (userType.isAdmin) {
+      editButton = IconButton.filled(
+        icon: const Icon(Icons.edit),
+        onPressed: () => navigator.push(const ScheduleEditor()),
+      );
+    }
     return Scaffold(
-      appBar: AppBar(
-        title: const Text('Livestream Schedule'),
-        actions: [
-          if (userType.isAdmin)
-            IconButton(
-              icon: const Icon(Icons.add),
-              onPressed: () => navigator.push(const ManageSchedule()),
-            ),
-        ],
-      ),
```

Comment:
This was the only page with both an app bar and a navigation bar, so I turned the `AppBar` button into a "floating action button".